### DISCUSSION TITLE
reset password is not showing

### DISCUSSION THREAD

**Discussion by mateusgalasso** [laravel/nova-issues#3491] [Q&A]
- Laravel Version: 8.12
- Nova Version: 3.27.0
- PHP Version: 8.0

### Description:
simply on a fresh install from scratch,  nova is not showing the forgot password link on the login screen
Its

**Comment by crynobone** [comment]
Unable to reproduce. 

https://user-images.githubusercontent.com/172966/128099886-4d5bf168-9a29-4fd5-8cf4-97cfbc4e2360.mp4

**Comment by mateusgalasso** [comment]
I published the views. Maybe that's...

**Comment by mateusgalasso** [comment]
Here the return is empty.
![image](https://user-images.githubusercontent.com/3990028/128100134-99ca5255-d5dc-46e7-bbc1-20c3c0b47d22.png)
But a dd() in NovaServideProvider returns true

**Comment by mateusgalasso** [comment]
I changed de path to /admin

**Comment by crynobone** [comment]
> I published the views. Maybe that's...

Please elaborate. `php artisan nova:publish` only publishes partials view. Not in any way related to this issue.

> I changed de path to /admin

Not related.

**Comment by mateusgalasso** [comment]
maybe this?
![image](https://user-images.githubusercontent.com/3990028/128100752-191f3454-b36c-4a3d-8d59-9100117c9e85.png)

```
class LoginController extends Controller
{
    /*
    |--------------------------------------------------------------------------
    | Login Controller
    |--------------------------------------------------------------------------
    |
    | This controller handles authenticating users for the application and
    | redirecting them to your home screen. The controller uses a trait
    | to conveniently provide its functionality to your applications.
    |
    */

    use AuthenticatesUsers, ValidatesRequests;

    /**
     * Create a new controller instance.
     *
     * @return void
     */
    public function __construct()
    {
        $this->middleware('nova.guest:'.config('nova.guard'))->except('logout');
    }

    /**
     * Show the application's login form.
     *
     * @return \Illuminate\Http\Response
     */
    public function showLoginForm()
    {
        return view('nova::auth.login');
    }

    /**
     * Log the user out of the application.
     *
     * @param  \Illuminate\Http\Request  $request
     * @return \Illuminate\Http\Response
     */
    public function logout(Request $request)
    {
        $this->guard()->logout();

        $request->session()->invalidate();

        return redirect($this->redirectPath());
    }

    /**
     * Get the post register / login redirect path.
     *
     * @return string
     */
    public function redirectPath()
    {
        return Nova::path();
    }

    /**
     * Get the guard to be used during authentication.
     *
     * @return \Illuminate\Contracts\Auth\StatefulGuard
     */
    protected function guard()
    {
        return Auth::guard(config('nova.guard'));
    }
```

**Comment by crynobone** [comment]
Not related.

**Comment by crynobone** [comment]
> simply on a fresh install from scratch, nova is not showing the forgot password link on the login screen

Nova will stubs all the proper configurations during `nova:install` command and you shouldn't have this issue

What you replying above indicates that you have customized the application (and the original statement is no longer valid). I don't see how this become Laravel Nova bugs, moving this to discussion.

**Comment by jcsoriano** [comment]
Hi @crynobone I am also experiencing what @stonkeep reported and I also changed the path to `/admin`

I am using Laravel Octane in case that is relevant.

The "Forgot Password" link appears on my local, but when I deploy it in prod (prod has Octane) `Nova::resetsPasswords()` returns false for some reason, even if `NovaServiceProvider@routes` has `->withPasswordResetRoutes()` in it

Would you happen to know why this happens?

If I do `Nova::resetsPasswords()` on tinker, it shows `true` but when I `dump(Nova::resetsPasswords())` on the login file it returns false

Tinker:
![image](https://user-images.githubusercontent.com/5937317/151979904-93d9e65e-17c1-41ea-8d36-70a343fc73cb.png)

Dump in Login.blade:
![image](https://user-images.githubusercontent.com/5937317/151979976-20119322-b51a-4a9b-b571-9a7fdb54e4ee.png)

The only place I see where `resetsPasswords` is being set back to false is in `InteractsWithEvents@flushState` does Octane have anything to do with that?
